### PR TITLE
Feat/alternative my first robot tutorial

### PR DESCRIPTION
**Pull request by charles-shingleton**
Added a host of small formatting tweaks and changes to contextual language in response to comments from Johanna's Test Drive (2020/09/30).

Original comments can be seen in https://coda.io/d/Robocorp-product-onboarding_dMZWftkSY7F/Improvement-ideas-for-onboarding_su_VO#Table-1_tucCi/r1

### CODE REVIEW

File: my-first-robot/04-running-our-robot.robot [hunk 1]
```diff
@@ -25,18 +25,31 @@ Find an Image from Google Images
...
-# We can do this by either:
-# * Clicking each cell and then the `▸` (from the icon toolbar) one after the other
-# * Running all the robot's cells in one go by clicking `▸▸` (this is known as 'running the notebook')
 #
-# Try either of these methods now... You'll see a 'Find Image' button appear as the robot has gathered all the resources it needs to operate after running the cells. Click the button! 
-# 
+# We have several options for running robots in Robocorp Lab:
+#
+# 1. **Click on each cell, and then the `▸`, one after the other:**  
+#     - The `▸` button is found in the icon toolbar
```
Comment (mikaukora) [inline my-first-robot/04-running-our-robot.robot:32]: There's a difference between 'run' from the play button and 'run' from the widget button which appears below the cell. This should be explained. Basically 'run' from the play button "reads cell to the memory" or even "compiles" it. Only when the widget button is pressed the cell is actually run.

File: my-first-robot/04-running-our-robot.robot [hunk 2]
```diff
@@ -25,18 +25,31 @@ Find an Image from Google Images
...
+#
+#
+# 2. **Running all the cells in an open `.robot` file in one go by clicking `▸▸`:**
+#     - The `▸▸` button is found in the icon toolbar
+#     - This is known as '*running the notebook*' and will ask whether you want to '*Restart Kernel?*' That's okay! You can click `Restart`
+#
+#
+# 3. **Running the main robot configured for a project by clicking `▸ Run Robot`:**
+#      - The `▸ Run Robot` button is found at the top-right of the Lab
+#      - The main robot for a project is configured in the `robot.yaml` file (We'll discuss this file in depth in a more advanced tutorial)
```
Comment (mikaukora) [inline my-first-robot/04-running-our-robot.robot:42]: Here the terminology is following:
Robot may contain one or multiple tasks. "Run Robot" runs the default task defined in robot.yaml (or asks which one to run if there are multiple tasks defined). 
Another feature of "Run Robot" is that it creates an environment as defined in conda.yaml.

File: my-first-robot/05-moving-our-robot-to-the-cloud.robot [hunk 3]
```diff
@@ -24,23 +24,30 @@ Find an Image from Google Images
...
-# 
-# The robot from this file is available in the Robocorp Cloud! Please note that any changes that we make to our robot will have to be republished.
-# 
+# A `Visit Robocorp Cloud` button should have appeared. Click it!
+#
+# All seems ready, let's publish our robot from here in Robocorp Lab! Look to the top right of the Lab window, in the tool bar, and click the `Publish to Robocorp Cloud` button.
+#
+# Now, select 'Example workspace', then select the 'Google Image Search' robot inside it and click `Publish`! It will ask if you want to overwrite the example robot in the Cloud with our new Lab's robot: we do, so select `Confirm`
+#
+# Our robot from this file is now available in the Robocorp Cloud! Please note that any changes that we make to our robot will have to be republished.
```
Comment (mikaukora) [inline my-first-robot/05-moving-our-robot-to-the-cloud.robot:47]: any changes that we make to our robot in _Lab_ will have to be republished.

**Review by mikaukora — COMMENTED** [review]
Added some comments.

**Comment by vermario** [comment]
Question @charles-shingleton  @mikaukora : should we delete the `deprecated.robot` file in the root folder? It seems it might be confusing for a new user?

**Comment by janipalsamaki** [comment]
We are removing the entrypoints and `package.yaml` from all the other examples. Maybe worth considering here, too?

**Comment by vermario** [comment]
( removed entrypoints and package.yaml, updated to rpafw 5.x in other examples: https://github.com/robocorp/example-activities/pull/58 )

**Comment by charles-shingleton** [comment]
@vermario Yeah, we can remove it with @mikaukora's agreement. The reason I left it in was I suggested that Hanna do some A/B testing for Test Drivers by asking them to open the old tutorial and compare it to the newer one they had just gone through. File had to stay there for that to happen but doesn't seem like it has been happening so am content to remove it.

@janipalsamaki Will update `rpafw` requirement to `5.x` and remove `entrypoints` and `package.yaml` too.

File: my-first-robot/03-keywords.robot [hunk 4]
```diff
@@ -32,7 +32,7 @@ Find Image
 # 2. Enter a search term in the searchbar
 # 3. Click 'Search'
 # 4. Wait for Google to find the images
-# 5. Choose an image they liked!
+# 5. Choose an image they like
 #
 # We can describe this step-by-step process a human would take to our robot using special commands that we made available when we added the *RPA.Browser* utility in our Robot's `***Settings***`. Here's how it looks...
```
[the remark this answers is not in the document]
Comment (charles-shingleton) [reply my-first-robot/03-keywords.robot:37]: It's too complicated to draw out here but suffice to say I've played around with the language in response to this. Will be in later updated request.

**Comment by mikaukora** [comment]
deprecated.robot can be removed now.

File: my-first-robot/04-running-our-robot.robot (hunk 1 above)
Comment (charles-shingleton) [reply my-first-robot/04-running-our-robot.robot:32]: Added additional context in each run method

File: my-first-robot/04-running-our-robot.robot (hunk 2 above)
Comment (charles-shingleton) [reply my-first-robot/04-running-our-robot.robot:42]: I've added the first two pieces of terminology but not sure about mentioning it 'creat[ing] an environment as defined in `conda.yaml`': might be bit complicated for the first-robot tutorial but can lump it in if it's very important to do so

**Review by mikaukora — APPROVED** [review]
This is good to go but needs to be merged in sync with cloud container update.

File: my-first-robot/conda.yaml [hunk 5]
```diff
@@ -5,4 +5,4 @@ dependencies:
   - python=3.7.5
   - pip=20.1
   - pip:
-      - rpaframework==4.2.*
+      - rpaframework==5.*
```
Comment (kariharju) [inline my-first-robot/conda.yaml:8]: This is already in the current pre-heats --> Good to go.

**Pull request merged**